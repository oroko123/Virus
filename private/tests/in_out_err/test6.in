znaczek 1000 1910 Poczta Polska
znaczek 3000 1910 Poczta Polska
znaczek 2199.23 1910 Poczta Polska
znaczek 217,15 1910 Poczta Polska
znaczek 400 1910 Poczta Polska
znaczek 1999.99 1910 Poczta Polska
znaczek 3500 1910 Poczta Polska
znaczek 211 1910 Poczta Polska
znaczek 4815 1910 Poczta Polska
znaczek 999 1910 Poczta Polska
1909 1911
