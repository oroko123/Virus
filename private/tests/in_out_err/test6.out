1910 Poczta Polska 211 znaczek
1910 Poczta Polska 217,15 znaczek
1910 Poczta Polska 400 znaczek
1910 Poczta Polska 999 znaczek
1910 Poczta Polska 1000 znaczek
1910 Poczta Polska 1999.99 znaczek
1910 Poczta Polska 2199.23 znaczek
1910 Poczta Polska 3000 znaczek
1910 Poczta Polska 3500 znaczek
1910 Poczta Polska 4815 znaczek
